setup.py

```diff
@@ -60,9 +60,10 @@ def install_requires():
 #####
 # Project-specific constants
 #####
-NAME = 'gordon'
+NAME = 'gordon-dns'
+PACKAGE_NAME = 'gordon'
 PACKAGES = find_packages(where='.')
-META_PATH = os.path.join(NAME, '__init__.py')
+META_PATH = os.path.join(PACKAGE_NAME, '__init__.py')
```

Comment:
Just so I fully understand this, the `name` arg in `setup.py`:

```py
setup(
    name=NAME,
    # snip
)
```
is therefore `gordon-dns`, and is what defines the name when `pip install`'ing, i.e. `pip install gordon-dns`. Then, the `packages` in `setup`:

```py
setup(
    # snip
    PACKAGES=find_packages(where='.'),
    # snip
)
```
defines the import namespace, as in `import gordon`?

(I'm looking at [PyYaml](https://github.com/yaml/pyyaml/blob/master/setup.py)'s `setup.py` for some guidance.)

Reply:
Yes - the `name` arg defines what the package will be registered as in pypi while the `packages` arg tells setuptools what "directory level packages" to include in the distribution, so I think that technically, the `gordon` directory name defines the import namespace (`find_packages` just collects it for setuptools).

Reply:
Oh, also, pip installing it locally gives:

```bash
(gordontmp) matt@work:~/code/gordon$ pip list
gordon-dns (0.0.1.dev0, /home/matt/code/gordon)
pip (9.0.1)
setuptools (38.2.4)
wheel (0.30.0)
(gordontmp) matt@work:~/code/gordon$ python
Python 3.7.0a0 (heads/master:122e88a835, Aug 31 2017, 15:49:09) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gordon
>>>
```

Pip knows it as `gordon-dns`, but `import gordon` works as expected.